libs/datasets/combined_datasets.py

```diff
@@ -208,10 +211,26 @@ def datasource_regions(
     CommonFields.TOTAL_TEST_ENCOUNTERS_VIRAL: [CovidTrackingDataSource],
     CommonFields.TEST_POSITIVITY_14D: [CMSTestingDataset],
     CommonFields.TEST_POSITIVITY_7D: [CDCTestingDataset],
-    CommonFields.VACCINES_DISTRIBUTED: [CANScraperStateProviders, CDCVaccinesDataset],
-    CommonFields.VACCINES_ADMINISTERED: [CANScraperStateProviders, CDCVaccinesDataset],
-    CommonFields.VACCINATIONS_INITIATED: [CANScraperStateProviders, CDCVaccinesDataset],
-    CommonFields.VACCINATIONS_COMPLETED: [CANScraperStateProviders, CDCVaccinesDataset],
+    CommonFields.VACCINES_DISTRIBUTED: [
+        CDCVaccinesCountiesDataset,
```

Comment:
Tiny bit nervous about including CDC county data at all since we haven't vetted it, but we can look at the snapshot.